### PR TITLE
e2e-test: refactor positronConsole

### PR DESCRIPTION
**Pull request by midleman**
This PR updates the PositronConsole POM and all related or impacted tests to use the Playwright API directly

### QA Notes
Full suite ran and passed. (Minus one test which is a separate issue to this PR)

@:console @:plots

### CODE REVIEW

File: test/automation/src/positron/fixtures/positronRFixtures.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class PositronRFixtures {
 		// We currently don't capture fixtures in the Playwright trace, so take a screenshot on failure
 		try {
 			await this.app.workbench.positronConsole.selectInterpreter(InterpreterType.R, desiredR, skipReadinessCheck);
-			await this.app.workbench.positronConsole.waitForReady('>', 2000);
+			await this.app.workbench.positronConsole.waitForReady('>', 40000);
```
Comment (testlabauto) [inline test/automation/src/positron/fixtures/positronRFixtures.ts:33]: Oh wow.  Was such a huge bump needed?  2 -> 40 s

Comment (midleman) [reply test/automation/src/positron/fixtures/positronRFixtures.ts:33]: I don't think it was actually using the timeout if I recall. I think it was a typo, a 2 second wait???

File: test/automation/src/positron/positronConsole.ts [hunk 2]
```diff
@@ -201,33 +196,46 @@ export class PositronConsole {
...
-			false,
-			(elements) => accept ? (!!elements && accept(elements.map(e => e.textContent))) : true);
-		return elements.map(e => e.textContent);
+
+	async waitForConsoleContents(
+		consoleText: string,
+		options: {
+			timeout?: number;
+			contain?: boolean;
+			matchCount?: number;
```
Comment (testlabauto) [inline test/automation/src/positron/positronConsole.ts:212]: so contain and matchCount have to be used together, right?  But both are optional?

Comment (midleman) [reply test/automation/src/positron/positronConsole.ts:212]: these are the defaults:
```
const { timeout = 15000, contain = true, matchCount = 1 } = options;
```
i just updated this snippet seconds before you commented to make it simpler. let me know if it still is not clear.

File: test/e2e/features/console/console-autocomplete.test.ts [hunk 3]
```diff
@@ -16,12 +15,9 @@ test.describe('Console Autocomplete', {
...
 		await app.workbench.positronConsole.pasteCodeToConsole('import pandas as pd');
 		await app.workbench.positronConsole.sendEnterKey();
-		await app.workbench.positronConsole.typeToConsole('df = pd.Dat');
 
-		const suggestionList = await app.workbench.positronConsole.getSuggestions();
-		if (suggestionList.length < 3) {
-			fail('Less than 3 suggestions found');
-		}
+		await app.workbench.positronConsole.typeToConsole('df = pd.Dat');
+		await expect(app.workbench.positronConsole.suggestionList).toHaveCount(8, { timeout: 15000 });
```
Comment (testlabauto) [inline test/e2e/features/console/console-autocomplete.test.ts:20]: cool.  and this is the same count on windows, too, right?
i could have sworn there were diffs but someone may well have fixed something

File: test/automation/src/positron/fixtures/positronRFixtures.ts (hunk 1 above)
Comment (midleman) [reply test/automation/src/positron/fixtures/positronRFixtures.ts:33]: updated per our convo 👍

File: test/e2e/features/console/console-autocomplete.test.ts (hunk 3 above)
Comment (midleman) [reply test/e2e/features/console/console-autocomplete.test.ts:20]: Yep! https://app.currents.dev/instance/aAPcTxXwztr6oZ6W

**Review by testlabauto — APPROVED** [review]
Love it.  Great simplifications.  Will be so nice to not have such a weird API for waitForConsoleContents any more.

**Pull request merged**